pkgs/development/compilers/zig/generic.nix

```diff
@@ -70,6 +70,7 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ andrewrk ] ++ lib.teams.zig.members;
     mainProgram = "zig";
+    broken = stdenv.isDarwin && (lib.versionOlder finalAttrs.version "0.12.0");
```

Comment:
I think adding a comment linking to the issue might be a good idea.

Reply:
Please do it.